apps/docs/app/globals.css

```diff
@@ -25,14 +23,12 @@ a {
 }
 
 main {
```

Comment:
Adding spacing for the header directly in the main  style feels weird. 

can't we put the body at 100vh instead?

Reply:
you should scope your change in the page instead